Vungle/build.gradle

```diff
@@ -1,4 +1,4 @@
-project.version = '6.8.1.0'
+project.version = '6.9.0.0'
```

Comment:
@anton-blashchuk-zoolatech Can we do something similar to what Jian did for AdMob, we need adapter version to reflect early access 6.9.0.0-early1, and make sure that adding alphanumeric suffix does not break anything on adapter/functionality side
https://github.com/Vungle/googleads-mobile-android-mediation/pull/82/commits/8c616620060cc1875ecdf543618e81ddc1d910f8